pylint_odoo/misc.py

```diff
@@ -172,6 +165,62 @@ def _check_{NAME_KEY}(self, module_path)
                         node.file = node_file_original
                         node.lineno = node_lineno_original
 
+    def visit_module(self, node):
+        self.wrapper_visit_module(node)
+
+    def add_message(self, *args, **kwargs):
+        version = (self.manifest_dict.get('version')
+                   if isinstance(self.manifest_dict, dict) else '')
+        match = re.match(r"(?P<version>\d+.\d+)\.\d+\.\d+\.\d+$", version)
+        name_check = [arg for arg in args]
+        if match and len(name_check) >= 1:
+            if not self._is_version_supported(match.group('version'),
+                                              name_check[0]):
+                return
+        return super(PylintOdooChecker, self).add_message(*args, **kwargs)
+
+    def _is_version_supported(self, version, name_check):
+
+        def version_to_tuple(version):
+            if '.' not in version and version.count('.') != 1:
+                return version
+            version = version.split('.')
+            return tuple([int(version[0]), int(version[1])])
```

Comment:
What about use `packaging.version.parse("8.0") == packaging.version.parse("8.0.1.0.1"[:3]) ## Result: True`?
And change the definition of valid versions `min_odoo_version` and `max_odoo_version ` to string instead of tuples.

Reply:
Done, I changed the definition of the version from tuple to string, and include the package `distutils.version.LooseVersion` to made more easy the comparation